server/webhook_jira.go

```diff
@@ -10,6 +10,47 @@ import (
 	"github.com/andygrunwald/go-jira"
 )
 
+const (
+	eventCreatedStr            = "event_created"
+	eventCreatedCommentStr     = "event_created_comment"
+	eventDeletedStr            = "event_deleted"
+	eventDeletedCommentStr     = "event_deleted_comment"
+	eventDeletedUnresolvedStr  = "event_deleted_unresolved"
```

Comment:
`eventDeletedUnresolvedStr` is currently not used. `eventDeletedUnresolvedStr` exists in the original masks so I ported it over here. I'm not sure what the difference is between it and `eventDeletedStr`.

Reply:
Because you had to point that out, makes me think that we need to rename it. :)  Or at least put this as a comment in the code.

Or.. Any reason we can reuse the existing enums? Just put the ones you're reusing as comments, if you want to keep them all together.

Reply:
Yes, I'm not a fan of the names. This was a way of introducing the string-based enums without affecting the legacy webhook. The new enums are assigned as strings, so they can be human-readable, and exist on the frontend and backend codebases without confusion.

There will need to be some small modifications to how the original masks are used in the legacy webhook, in order to migrate to the new string-based enums. That should probably be a part of this PR as a migration, but the main logic of how the new enums work is the most important part of this PR. I will add the fix in to make both versions of webhooks use the same enums before this PR gets merged.